backend/codesirius/codesirius/kafa_producer.py

```
@@ -0,0 +1,31 @@
+
+class KafkaProducerSingleton:
+    _instance = None
+
+    @classmethod
+    def get_instance(cls):
+        if cls._instance is None:
+            cls._instance = Producer(kafka_conf)
+        return cls._instance
+
+    @classmethod
+    def produce_message(cls, topic, value, callback=None):
```

Ruff (0.8.2)

Missing return type annotation for classmethod `produce_message`

Add return type annotation: `None`

(ANN206)